nautobot/extras/models/groups.py

```diff
@@ -393,29 +396,8 @@ def get_initial(self):
         This is intended for use to populate the dynamically-generated filter form created by
         `generate_filter_form()`.
         """
-        filter_fields = self.get_filter_fields()
```

Comment:
Can you please go ahead and remove the `FIXME` comment above here? Your changes obviate it. Thank you!